### PR TITLE
Make TOK a ubyte

### PR DESCRIPTION
**Pull request by WalterBright**
Achievement unlocked! Finally got the number of TOKs under 256 so we can make it a ubyte again.

### CODE REVIEW

File: src/dmd/tokens.d [hunk 1]
```diff
@@ -1113,7 +1113,7 @@ nothrow:
         return toString(value).ptr;
     }
 
-    static const(char)* toChars(ushort value)
+    static const(char)* toChars(ubyte value)
```
Comment (ibuclaw) [inline src/dmd/tokens.d:1116]: Does this need updating too in tokens.h?

Comment (WalterBright) [reply src/dmd/tokens.d:1116]: Redefining `TOK` does that.

Comment (ljmf00) [reply src/dmd/tokens.d:1116]: Then you maybe need to update the function signature there too

Comment (RazvanN7) [reply src/dmd/tokens.d:1116]: Since the testsuite is passing, I guess this is good to go?

**Review by RazvanN7 — APPROVED** [review]

File: src/dmd/tokens.d (hunk 1 above)
Comment (ibuclaw) [reply src/dmd/tokens.d:1116]: @WalterBright then this function signature should accept a `TOK` instead then?

Comment (ibuclaw) [reply src/dmd/tokens.d:1116]: Ohhh, this function is dead code. Kill it without mercy!

Comment (RazvanN7) [reply src/dmd/tokens.d:1116]: We can merge this and then delete it in a subsequent PR.

Comment (ibuclaw) [reply src/dmd/tokens.d:1116]: @RazvanN7 I'm waiting for your subsequent PR, it's been 2 minutes and still isn't here yet. ;-)

Comment (RazvanN7) [reply src/dmd/tokens.d:1116]: Well, it took some time to make sure that all the appropriate headers are updated: https://github.com/dlang/dmd/pull/13436

**Pull request merged**